rasa/core/channels/console.py

```diff
@@ -9,37 +9,37 @@
 from async_generator import async_generator, yield_
 from prompt_toolkit.styles import Style
 
-import rasa.cli.utils
+import rasa.cli.utils as cliutils
+from rasa.cli.utils import button_to_string, element_to_string
```

Comment:
same here, we already have the import above so just use `cliutils.button_to_string` (though `cli_utils` is more legible)